src/components/Participants/Participants.js

```diff
@@ -5,21 +5,101 @@
  * of the MIT license.  See the LICENSE.txt file for details.
  */
 
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useSelector } from 'react-redux';
 
 import Participant from '../Participant';
 
+/*
+ * See setParticipantVars
+ */
+function getCssConfig(element, property, def) {
+  let value = element.style.getPropertyValue(property);
+
+  if (value === "") {
+    return def;
+  } else {
+    return Number(value);
+  }
+}
```

Comment:
NP: I'd use a ternary operator over here. A matter of taste.

```js
return value === "" ? def : Number(value)
```

Reply:
Done.